orion/model.py

```diff
@@ -111,9 +115,17 @@ class Pipeline(Document, MongoUtils):
     created_by = fields.StringField()
 
 
-class Datarun(Document, MongoUtils):
-    dataset = fields.ReferenceField(Dataset)
+class Experiment(Document, MongoUtils):
+    name = fields.StringField(required=True)
```

Comment:
Do we need a name for the experiment?
I think we could skip this field and just have the internal `_id` and the `insert_time`.